recipes-init/openrc/openrc_0.45.2.bb

```diff
@@ -21,17 +23,36 @@ PACKAGECONFIG[selinux] = "-Dselinux=enabled,-Dselinux=disabled,libselinux"
 PACKAGECONFIG[usrmerge] = "-Drootprefix=/usr,-Drootprefix=/"
 PACKAGECONFIG[zsh-completions] = "-Dzsh-completions=true,-Dzsh-completions=false"
 
+openrc_sbindir = "${@bb.utils.contains('PACKAGECONFIG', 'usrmerge', '${sbindir}', '${base_sbindir}', d)}"
+openrc_libdir = "${@bb.utils.contains('PACKAGECONFIG', 'usrmerge', '${libdir}', '${base_libdir}', d)}"
+
 EXTRA_OEMESON += " \
     -Dos=Linux \
     -Dpkg_prefix=${prefix} \
+    -Dsysvinit=true \
 "
 
+USE_VT ?= "1"
+
+add_getty() {
+    ln -snf getty ${D}${OPENRC_INITDIR}/getty.$1
+    ln -snf ${OPENRC_INITDIR}/getty.$1 ${D}${sysconfdir}/runlevels/default
+    echo "baud=$2" > ${D}${OPENRC_CONFDIR}/getty.$1
+    if [ -n "$3" ]; then
+        echo "term_type=$3" >> ${D}${OPENRC_CONFDIR}/getty.$1
+    fi
+}
+
 do_install:append() {
     # Default sysvinit doesn't do anything with keymaps on a minimal install so
     # we're not going to either.
     rm ${D}${sysconfdir}/runlevels/*/keymaps
 
-    install -m 755 ${WORKDIR}/volatiles.initd ${D}${OPENRC_INITDIR}/volatiles
+    for svc in getty volatiles; do
+        install -m 755 ${WORKDIR}/$svc.initd ${D}${OPENRC_INITDIR}/$svc
```

Comment:
It's somewhat of a personal preference, but as a former Gentoo dev, I had wrapping bash variables in `{}` drilled into me.  It would be consistent with the rest of this layer:
```
$ rg --glob '*.bb*' '\$\{' | wc -l
235
$ rg --glob '*.bb*' '\$[^{(]'  | wc -l
7
```

Reply:
Sure, will fix.